docs/source/tracking/server.rst

```diff
@@ -0,0 +1,247 @@
+  [2023-11-01 10:28:12 +0900] [28553] [INFO] Booting worker with pid: 28553
+  [2023-11-01 10:28:12 +0900] [28555] [INFO] Booting worker with pid: 28555
+  [2023-11-01 10:28:12 +0900] [28558] [INFO] Booting worker with pid: 28558
+  ...
+
+There are many options to configure the server, refer to :ref:`Configure Server <configure-server>` for more details.
+
+.. important:: 
+  The server listens on http://localhost:5000 by default and only accepts
+  connections from the local machine. To let the server accept connections
+  from other machines, you will need to pass ``--host 0.0.0.0`` to listen on
+  all network interfaces (or a specific interface address).
+
+.. _logging_to_a_tracking_server:
+
+Logging to a Tracking Server
+============================
+
+Once you started the tracking server, you can connect your local clients by set the ``MLFLOW_TRACKING_URI`` environment variable to the 
+server's URI, along with its scheme and port (for example, ``http://10.0.0.1:5000``) or call :py:func:`mlflow.set_tracking_uri`.
+
+The :py:func:`mlflow.start_run`, :py:func:`mlflow.log_param`, and :py:func:`mlflow.log_metric` calls
+then make API requests to your remote tracking server.
+
+  .. code-section::
+
+    .. code-block:: python
+
+        import mlflow
+
+        remote_server_uri = "..."  # set to your server URI
+        mlflow.set_tracking_uri(remote_server_uri)
+        # Note: on Databricks, the experiment name passed to mlflow_set_experiment must be a
+        # valid path in the workspace
+        mlflow.set_experiment("/my-experiment")
+        with mlflow.start_run():
+            mlflow.log_param("a", 1)
+            mlflow.log_metric("b", 2)
+
+    .. code-block:: R
+
+        library(mlflow)
+        install_mlflow()
+        remote_server_uri = "..." # set to your server URI
+        mlflow_set_tracking_uri(remote_server_uri)
+        # Note: on Databricks, the experiment name passed to mlflow_set_experiment must be a
+        # valid path in the workspace
+        mlflow_set_experiment("/my-experiment")
+        mlflow_log_param("a", "1")
+
+.. _configure-server:
+
+Configure Server
+================
+This section describes how to configure the tracking server for some common use cases. Please run ``mlflow server --help`` for the full list of command line options.
+
+Backend Store
+-------------
+By default, the tracking server logs runs metadata to the local filesystem under ``./mlruns`` directory.
+You can configure the different backend store by adding ``--backend-store-uri`` option:
+
+Example
+.. code-block:: bash
+
+    mlflow server --backend-store-uri sqlite:///my.db
+
+This will create a SQLite database ``my.db`` in the current directory, and logging requests from clients will be pointed to this database.
+
+.. note::
+  If you are using SQL-based backend store, artifact store URI has to be specified otherwise the server will fail to start (because we can't store artifacts to SQL database).
```

Comment:
```suggestion
  If you are using a SQL-based backend store, the artifact store URI has to be specified, otherwise the server will fail to start (because we can't store artifacts to a SQL database).
```